libraries/standard/http/include/http_client.h

```diff
@@ -53,12 +52,12 @@
  * @brief Set this flag to disable automatically writing the Content-Length
  * header. 
  */
-#define HTTP_REQUEST_DISABLE_AUTO_CONTENT_LENGTH_FLAG   0x2U
-
+#define HTTP_REQUEST_DISABLE_CONTENT_LENGTH_FLAG   0x2U
 
 /**
  * Flags for #HTTPResponse_t.flags.
- * These flags are in #HTTPResponse_t.flags after #HTTPClient_Send() returns.
+ * These flags are updated in #HTTPResponse_t.flags after #HTTPClient_Send()
```

Comment:
(Nitpick) These flags are _populated_ in #HTTPResponse_t.flags _by the_ #HTTPClient_Send() function.
(_After #HTTPClient_Send returns_ is misleading..)

Reply:
populated does sounds better